math/finding_number_of_digits_in_a_number.cpp

```diff
@@ -40,27 +43,59 @@ uint64_t finding_number_of_digits_in_a_number(uint64_t n) {
     return count;
 }
 
+/**
+ * @brief This function finds the number of digits
+ * in constant time using logarithmic function
+ * TC: O(1)
+ * @param n the number to check its digits
+ * @returns the digits count
+ */
+double finding_number_of_digits_in_a_number_using_log(double n) {
+    // to handle the negative numbers
+    if (n < 0) {
+        n = -n;
+    }
+    // log(0) is undefined
+    else if (n == 0) {
+        // handling the value 0
+        return 0;
+    }
```

Comment:
Use guard if you already know what to return. Looking at your second `if` statement, it's clear that you already know what to `return` so use that as guard instead of `else if`.

 ```suggestion
    if (n == 0) {
        // handling the value 0
        return 0;
    }
    if (n < 0) {
        n = -n;
    }
```

Reply:
Please try to not @ mention people. It creates a lot of unnecessary noise for maintainers 🙏🏼